servers/orbitmc/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "orbitmc",
+    "name": "OrbitMc",
+    "description": "Place for ASIAN Pvpers",
+    "addresses": [
+        "orbitmc.xyz"
+
+    ],
+    "primaryAddress": "mc.orbitmc.xyz",
+    "minecraftVersions": [
+        "1.7.*",
+        "1.8.*"
+
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#FFD700",
+    "secondaryColor": "#FFFF00",
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "AS",
+    "regions": [
+        "AS"   
+    ],
+    "gameTypes": [
+        "PvP",
+        "Bedwars"

```

Comment:
Bedwars is not valid please change it with one of the following: 

['PvP', 'PvE', 'HCF', 'Factions', 'Minigames', 'Skyblock', 'Parkour', 'UHC', 'Hardcore', 'Survival', 'Open World', 'Prison', 'Creative', 'Roleplay', 'Adventure']

Reply:
is that it? i did it now please merge